src/msgwindow.c

```diff
@@ -112,6 +117,40 @@ void msgwin_set_messages_dir(const gchar *messages_dir)
 }
 
 
+GdkColor load_color(gchar *color_name) {
```

Comment:
IIUC standard x86-64 ABI says up to 16 bytes can be returned in 2 registers, so probably no real copying takes place.